site/content/en/docs/handbook/mount.md

```diff
@@ -74,7 +74,7 @@ Some hypervisors, have built-in host folder sharing. Driver mounts are reliable
 | VirtualBox | Windows | C://Users | /c/Users |
 | VMware Fusion | macOS | /Users | /mnt/hgfs/Users |
 | KVM | Linux | Unsupported | |
-| HyperKit | Linux | Unsupported (see NFS mounts) | |
+| HyperKit | macOS | Unsupported | |
```

Comment:
hyperkit DOES support nfs mounts.
instead of deleting it lets add right documentation for it.

```
$ minikube start --help | grep nfs
      --nfs-share=[]: Local folders to share with Guest via NFS mounts (hyperkit driver only)
      --nfs-shares-root='/nfsshares': Where to root the NFS Shares, defaults to /nfsshares (hyperkit driver only)
```

Reply:
Hi @medyagh, I have added in the appropriate docs for the `nfs flags`.